compiler/cpp/src/thrift/generate/t_erl_generator.cc

```diff
@@ -984,10 +988,13 @@ void t_erl_generator::export_types_string(string name, int num) {
 }
 
 void t_erl_generator::export_function(t_function* tfunction, string prefix) {
-
+  t_struct::members_type::size_type num = tfunction->get_arglist()->get_members().size();
+  if (num > std::numeric_limits<int>().max()) {
```

Comment:
Yes, I just finished a suite of batch files to build zlib, openssl, libevent, and thrift for VS 2010 through 2015, win32/x64, debug/release.  I started running into these signed/unsigned issues today and I am working through resolving it.  The acceptance criteria for this issue is that I can build all combinations with zero warnings - we'll see how close I get.